index.html

```diff
@@ -4,47 +4,185 @@
+                        <li><a href="#">JavaScript</a></li>
+                </ul>
+                <ul class="btn">
+                    <li><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="card">
+                <div>
+                    <img src="assets/mps-bg-2.png" alt=""/>
+                </div>
+                <div class="card-title">
+                    <h2>
+                        Multi-Post <br />
+                        Stories
+                    </h2>
+                    <div class="caption">
+                        <h3>CANOPY </h3>
+                        <img src="assets/dot.png" alt="" />
+                        <h4>BackEndDev</h4>
+                        <img src="assets/dot.png" alt="" />
+                        <h4>2015</h4>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="styled-links">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                </ul>
+                <ul class="btn">
+                    <li><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="profile-card">
+                <div class="header-text">
+                    <img src="assets/me.png" alt=""/>
+                </div>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <div class="connect-img"><img src="assets/connect.png" alt=""/></div>
+                <div class="social-media-icons">
+                    <ul>
+                        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                        <li><a href="#"><i class="fab fa-linkedin"></i></a></li>
+                        <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
+                        <li><a href="#"><i class="fab fa-github"></i></a></li>
+                        <li><a href="#"><i class="fab fa-angellist"></i></a></li>
+                      </ul>
+                </div>
+                <ul class="btn">
+                    <li><a href="#">Get Resume</a></li>
+                </ul>
+                <p>&nbsp;</p>
+                <div class="specials"><img src="assets/Languages.png" alt=""/><a href="#"><i class="fa fa-angle-down icons" aria-hidden="true"></i></a></div>
+                <br />
+                <div>
+                    <img src="assets/icon-js.png" alt=""/> <br />
+                    <img src="assets/icon-html.png" alt="" /> <br />
+                    <img src="assets/icon-css.png" alt="" />
```

Comment:
- [ ] This is a list of languages and you are required to enclose this in a `ul` tag with each language wrapped in a `li` tag